src/librustc_mir/interpret/intern.rs

```diff
@@ -352,23 +352,24 @@ where
             inside_unsafe_cell: false,
         }
         .visit_value(mplace);
-        if let Err(error) = interned {
-            // This can happen when e.g. the tag of an enum is not a valid discriminant. We do have
-            // to read enum discriminants in order to find references in enum variant fields.
-            if let err_ub!(ValidationFailure(_)) = error.kind {
-                let err = crate::const_eval::error_to_const_error(&ecx, error);
-                match err.struct_error(
-                    ecx.tcx,
-                    "it is undefined behavior to use this value",
-                    |mut diag| {
-                        diag.note(crate::const_eval::note_on_undefined_behavior_error());
-                        diag.emit();
-                    },
-                ) {
-                    ErrorHandled::TooGeneric
-                    | ErrorHandled::Reported(ErrorReported)
-                    | ErrorHandled::Linted => {}
-                }
+        // We deliberately *ignore* interpreter errors here.  When there is a problem, the remaining
```

Comment:
I'm not sure if we hit all the allocations. If this error is happening while looking at the first field of a struct, and that errors, the second field is never looked at. So if the second field contains a relocation, I'm guessing that we won't actually intern that relocation. Validation can handle that though, I think, so just the comment here that needs updating

Reply:
> if the second field contains a relocation, I'm guessing that we won't actually intern that relocation.

We will, in the leftover phase.